javalib/src/main/scala/java/util/Date.scala

```diff
@@ -0,0 +1,40 @@
+package java.util
+
+/** Ported from Scala JS and Apache Harmony
+ * - omits deprecated methods
+ * - immutable, setTime not implemented
+ * - toString not jdk compatible
+ */
+class Date(milliseconds: Long)
```

Comment:
I don't have any statistic on the usage of the `java.util.Date` yet, but it would be nice to be on par with Scala.js in terms of the number of methods we support.